### PR TITLE
AR-11299 Updated docs for writing custom statuses

### PR DESCRIPTION
**Pull request by ryan-novak-nextech**

### CODE REVIEW

File: source/includes/_appointmentstatus.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Searches for all appointment statuses matching the given search criteria. See [https://www.hl7.org/fhir/search.html](https://www.hl7.org/fhir/search.html) for instructions on formatting search criteria.
+
+#### HTTP Request 
+`GET /appointment-status?{parameters}`
+
+#### Parameters
+| Name | Located in | Description | Required | Initial Version |
+| ---- | ---------- | ----------- | -------- | --------------- |
+| identifier | query | The unique identifier for a single appointment status | N | _15.9_ |
+| name | query | The name of the appointment status | N | _15.9_ |
```
Comment (JakeDriscoll) [inline source/includes/_appointmentstatus.md:43]: Should put an example here of a search by name

**Review by JakeDriscoll — CHANGES_REQUESTED** [review]
Need to have an additional sample for doing an appointment status query other than by `appointment-status/{id}`

File: source/includes/_appointmentstatus.md (hunk 1 above)
Comment (ryan-novak-nextech) [reply source/includes/_appointmentstatus.md:43]: I added another example. Also, I noticed as I was searching for other examples that appointment type doesn't quite match the formatting of some of our other sections, so I updated the formatting of status to be a bit closer to what we have in most of the rest of the docs.

**Review by JakeDriscoll — APPROVED** [review]
Looks good

**Pull request merged**